components/IscnRegisterForm.vue

```diff
@@ -826,7 +826,7 @@ export default class IscnRegisterForm extends Vue {
   authorName: string = ''
   authorUrl: string[] = []
   authorWalletAddress: string[] = []
-  sentArweaveTransactionHashes = new Map<string, string>()
+  sentArweaveTransactionHashes = new Map<string, any>()
```

Comment:
if we already know it the `any` is a object with 2 optional key `transactionHash` and `arweaveId` we might as well define it